wordpress-mobile/WordPress-Android
# Update dashboard card domain to v2

Updating dashboard card domain ui to match latest v2

Fixes #18280 

| Light | Dark | Bold text
|--------|--------|--------|
| ![Screenshot_20230426_103917](https://user-images.githubusercontent.com/990349/234437709-1efbe978-104f-4724-8253-e9f503d86b03.png) | ![Screenshot_20230426_103937](https://user-images.githubusercontent.com/990349/234437725-88d92eb8-d2ef-44ce-8cb1-db4b40300109.png) | ![Screenshot_20230426_104020](https://user-images.githubusercontent.com/990349/234437738-512bd4a5-a8dd-4afd-9a39-de0532a2c5d3.png) |

To test:

- Launch Jetpack/WordPress app
- Verify **domain card** appears as shown above on dashboard
- Switch to dark/night mode
- Verify **domain card** matches the dark version

Additionally use the UI changes checklist

- **Tap** on **domain card**
- **Verify** that it launches domains search
- Tap on overflow (three dots) menu
- **Verify** it pops up menu with **Hide this** option
- Tap on Hide this
- **Verify**, the card gets hidden 

Also, do some relevant test from the UI Changes Checklist below

>** Note **
> The browser image needs to be updated, waiting for a final version.  Will update before merging

## Regression Notes
1. Potential unintended areas of impact
None

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Tested manually

3. What automated tests I added (or what prevented me from doing so)
Existing unit tests

PR submission checklist:


UI Changes testing checklist:
